dotnet/src/SemanticKernel/Kernel.cs

```diff
@@ -143,6 +143,10 @@ public void RegisterMemory(ISemanticTextMemory memory)
         this._memory = memory;
     }
 
+    /// <inheritdoc/>
+    public Task<SKContext> RunAsync(ISKFunction skFunction, CancellationToken cancellationToken)
```

Comment:
Same comment - I think this would be (temporarily) ambiguous with the params... overload, when no token is passed.
I can try it and see if the compiler squawks.

Reply:
Nevermind. Seems fine! Added